### ISSUE TITLE
Line transformation

### ISSUE THREAD

**Issue by yj7o5** [dehru/grunt-copy-part-of-file#9]
Hey Man,

Wouldn't be nice to also have a way to transform a particular line or each of the lines that are being copied.

so for ex:
```
copy_part_of_files: {
     options: {},
     transform: (line:string) => [transformed] line
}
```

this could be useful in cases where lets say we have a Constants file in our backend proj. and another constants.js file in the frontend wanted to keep them in sync. Now if it were to be a .cs file we may need to transform the line that is "parse-able" in js. 
like 
```
public const string SomethingAsFoo = "SomethingAsFoo"
-
SomethingAsFoo: "SomethingAsFoo" 
```
but there are many possibilities.

**Comment by dehru** [comment]
Yeah, that's a nice idea.  You have any interest in adding that?  I'm not using grunt much anymore, I've switched to gulp on new projects.

**Comment by yj7o5** [comment]
Sorry been busy quite a while.

ok yea ... will send you a PR soon ... once I get some time from school and work.

**Comment by yj7o5** [comment]
Alright have a look on the PR ..

**Comment by dehru** [comment]
Sorry, I’ve been sick.  I will try to take a look ASAP.
